saltpylint/strings.py

```diff
@@ -151,8 +134,7 @@ def visit_call(self, node):
                         )
 
                     if BAD_FORMATTING_SLOT.findall(inferred.value):
-                        if self.config.un_indexed_curly_braces_always_error or \
-                                sys.version_info[:2] < (2, 7):
+                        if sys.version_info[:2] < (2, 7):
```

Comment:
Any Py2 code or logic can just be removed

Reply:
removed and add back in un_indexed_curly_braces_always_error check